.travis.yml

```diff
@@ -0,0 +1,14 @@
+language: go
+dist: trusty
+
+go:
+  - 1.7
+
+install:
+  - go get github.com/Sirupsen/logrus
```

Comment:
We prefer vendoring. This pr is just to get some simple testing put in place. We will work on getting this project more aligned with the our general standard practices.